### PR TITLE
cb256 Implement the loan amount step 9

### PR DESCRIPTION
**Pull request by Brandonz07**
image

### CODE REVIEW

**Comment by novadeveloper14** [comment]
solve conflicts

File: src/components/data/Fieldset/styles.ts [hunk 1]
```diff
@@ -11,7 +11,7 @@ interface IStyledContainerFieldset {
 export const StyledContainerFieldset = styled.div<IStyledContainerFieldset>`
   cursor: pointer;
   box-sizing: border-box;
-  overflow-x: hidden;
+  overflow-x: ${({ $hasOverflow }) => $hasOverflow && "visible"};
```
Comment (novadeveloper14) [inline src/components/data/Fieldset/styles.ts:14]: the app is breaking

image

File: src/pages/add-prospect/interface.tsx [hunk 2]
```diff
@@ -14,6 +14,7 @@ import {
 } from "./types";
 import { StyledContainerAssisted } from "./styles";
 import { RequirementsNotMet } from "./steps/requirementsNotMet";
+import { LoanAmount } from "./steps/loanAmount";
 
 interface StepDetails {
```
Comment (novadeveloper14) [inline src/pages/add-prospect/interface.tsx:19]: Any other interface or type other than the component's props should go in a type.ts

File: src/pages/add-prospect/interface.tsx
Comment (novadeveloper14) [inline src/pages/add-prospect/interface.tsx:1]: In the project we are organizing the imports in the following way:

External imports (from inubekit, icons, hooks... any library external to the project)
////////space////////
Internal imports with aliases (imports with @alias currently the ones we have in the vite.config.ts file)
///////space//////
Internal imports without aliases (such as "./steps/extraordinaryInstallments")

This must be applied to all files that are created or modified. Whenever you see a file that does not comply with this rule, please modify it.

File: src/pages/add-prospect/interface.tsx [hunk 3]
```diff
@@ -49,6 +50,25 @@ export function AddProspectUI(props: AddPositionUIProps) {
 
   const smallScreen = useMediaQuery("(max-width:880px)");
 
+  const planData = [
```
Comment (novadeveloper14) [inline src/pages/add-prospect/interface.tsx:53]: This should go in a configuration file or be mock simulated. evaluate with camilo

File: src/pages/add-prospect/steps/loanAmount/index.tsx [hunk 4]
```diff
@@ -0,0 +1,106 @@
+import { useState } from "react";
```
Comment (novadeveloper14) [inline src/pages/add-prospect/steps/loanAmount/index.tsx:1]: In the project we are organizing the imports in the following way:

External imports (from inubekit, icons, hooks... any library external to the project)
////////space////////
Internal imports with aliases (imports with @alias currently the ones we have in the vite.config.ts file)
///////space//////
Internal imports without aliases (such as "./steps/extraordinaryInstallments")

This must be applied to all files that are created or modified. Whenever you see a file that does not comply with this rule, please modify it.

**Review by novadeveloper14 — CHANGES_REQUESTED** [review]

**Review by novadeveloper14 — APPROVED** [review]

File: src/mocks/add-prospect/request-value/requesvalue.mock.ts [hunk 5]
```diff
@@ -0,0 +1,20 @@
+import { IRequestVelue } from "@services/types";
+
+export const mockRequestValue: IRequestVelue[] = [
```
Comment (carin91) [inline src/mocks/add-prospect/request-value/requesvalue.mock.ts:3]: change the name to mockPaymentChannel

File: src/pages/add-prospect/steps/loanAmount/index.tsx [hunk 6]
```diff
@@ -0,0 +1,120 @@
...
+            {dataAmount.ordinaryPayment}
+          </Text>
+          <Select
+            id="paymentPlan"
+            options={requestValue || []}
+            placeholder={dataAmount.selectOption}
+            name="paymentPlan"
+            onChange={onChangeSelect}
+            value={form["paymentPlan"]}
+            fullwidth
```
Comment (carin91) [inline src/pages/add-prospect/steps/loanAmount/index.tsx:114]: avoid coercion

**Review by carin91 — APPROVED** [review]

**Pull request merged**